paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodRowButton.kt

```diff
@@ -216,7 +220,9 @@ private fun RowButtonRadioOuterContent(
             selected = isSelected,
             onClick = onClick,
             enabled = isEnabled,
-            modifier = Modifier.align(Alignment.CenterVertically).size(20.dp),
+            modifier = Modifier
```

Comment:
I can revert all these if you want, I just did auto format in the IDE, and it changed these too.